pkg/csi/service/node.go

```diff
@@ -104,11 +104,13 @@ func (driver *vsphereCSIDriver) NodeStageVolume(
 	}
 
 	if volCap == nil {
-		return nil, status.Error(codes.InvalidArgument, "Volume capability not provided")
+		return nil, logger.LogNewErrorCode(log, codes.InvalidArgument,
+			"Volume capability not provided")
 	}
 	caps := []*csi.VolumeCapability{volCap}
 	if err := common.IsValidVolumeCapabilities(ctx, caps); err != nil {
-		return nil, status.Errorf(codes.InvalidArgument, "Volume capability not supported. Err: %+v", err)
+		return nil, logger.LogNewErrorCodef(log, codes.InvalidArgument,
+			"Volume capability not supported. Err: %+v", err)
```

Comment:
Volume -> volume

Same comment applies to some of the other message changes.

Reply:
@xing-yang Do you mind if I address this in separate reviews? I prefer to fix one anti pattern at a time, which is easier for reviewers.

I do fix multiple anti-patterns together, if the change is small. But, I feel we need to get closure on one anti-pattern first, before moving to another.

Of course, if you insist, I will sweep through the file in the same change.

Reply:
That's fine.